### DISCUSSION TITLE
Welcome to charm4py Discussions!

### DISCUSSION THREAD

**Discussion by matthiasdiener** [charmplusplus/charm4py#185] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by doncandela** [comment]
I'm in the physics department at UMass, Amherst, and I have a compute-intensive code to simulate flowing granular media (a bit like LAMMPS).  I have succeeded in parallelizing the code using charm4py getting speedups on multi-core desktop PCs and on single nodes at our HPC center, although the latter  do not scale very well with the number of cores.  From the docs it seems I should be using MPI rather than the default TCP and that I will need that to move to a multi-node scaling.  But so far I can't figure out how - the docs say do
$ pip3 install [--mpi] charm4py --no-binary charm4py
but this tells me --mpi is an unknown option for pip.

**Reply by matthiasdiener** [reply]
Hi @doncandela, I believe the command should be:

```
$ pip3 install --install-option="--mpi" charm4py --no-binary charm4py
```
